src/components/Xmark.jsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+const Svg = styled.svg`
+  position: absolute;
+  font-weight: 600;
+  top: 15px;
+  right: 15px;
+  cursor: pointer;
+`;
+
+export default function Xmark({ setModalClick }) {
+  return (
+    <Svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      onClick={() => {
+        setModalClick(false);
+      }}
```

Comment:
setModalClick 모달 컴포넌트와 밀접하게 관련이 있어보이는데, 간단하게 onClick이라고 하면 좋겠네요. 그러면 외부 컴포넌트에서  `        setModalClick(false);`을 처리할 거에요.

```suggestion
      onClick={onClick}
```